java gr 8/homework/07homework7_obj_containers/src/main/java/ev/purchase/program/domain/EvCar.java

```diff
@@ -0,0 +1,107 @@
+package ev.purchase.program.domain;
+
+/**
+ * Created by Narci on 8/26/2017
+ */
+public class EvCar {
+
+    float price;
```

Comment:
Why are some members private and other default? Please choose the correct approach.